client/src/modules/stock/lots-duplicates/duplicate-lots.html

```diff
@@ -0,0 +1,104 @@
+            <span class="fa fa-bars"></span> <span class="hidden-xs" translate>FORM.LABELS.MENU</span> <span class="caret"></span>
+          </a>
+
+          <ul uib-dropdown-menu role="menu" class="dropdown-menu-right">
+            <li role="menuitem">
+              <a href data-method="configure" ng-click="DupeLotsCtrl.openColumnConfigModal()">
+                <i class="fa fa-columns"></i> <span translate>FORM.LABELS.COLUMNS</span>
+              </a>
+            </li>
+
+            <li role="separator" class="divider"></li>
+            <li role="menuitem">
+              <a href data-method="save-state" ng-click="DupeLotsCtrl.saveGridState()">
+                <i class="fa fa-save"></i> <span translate>FORM.BUTTONS.SAVE_GRID_CONFIGURATION</span>
+              </a>
+            </li>
+
+            <li role="menuitem">
+              <a href data-method="clear-state" ng-click="DupeLotsCtrl.clearGridState()">
+                <i class="fa fa-close"></i> <span translate>FORM.BUTTONS.CLEAR_GRID_CONFIGURATION</span>
+              </a>
+            </li>
+
+          </ul>
+        </div>
+      </div>
+      <!--
+      <bh-search-button on-click="DupeLotsCtrl.search()"></bh-search-button>
+
+      <div class="toolbar-item">
+        <div class="btn-group" uib-dropdown>
+          <a class="btn btn-default"
+            ng-class="{ 'btn-info' : DupeLotsCtrl.grouped }"
+            ng-click="DupeLotsCtrl.toggleGroup(DupeLotsCtrl.selectedGroup.value)"
+            ng-disabled="!DupeLotsCtrl.selectedGroup.label">
+            <span>
+              <i class="fa fa-object-group"></i>
+              <span ng-hide="DupeLotsCtrl.selectedGroup.label" translate>STOCK.GROUPING</span>
+              <span
+                ng-show="DupeLotsCtrl.selectedGroup.label"
+                translate>
+                {{ DupeLotsCtrl.selectedGroup.label }}
+              </span>
+            </span>
+          </a>
+          <a class="btn btn-default" uib-dropdown-toggle>
+            <span class="caret"></span>
+          </a>
+          <ul class="dropdown-menu" uib-dropdown-menu role="menu" aria-labelledby="single-button">
+            <li ng-repeat="option in DupeLotsCtrl.groupingBox">
+              <a href ng-click="DupeLotsCtrl.selectGroup(option)">
+                <span translate>{{ option.label }}</span>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    -->
+      <!-- inline filter -->
+      <!--
+      <bh-filter-toggle
+        on-toggle="DupeLotsCtrl.toggleInlineFilter()">
+      </bh-filter-toggle>
+      -->
+    </div>
+  </div>
+</div>
+<!--
+<div class="flex-util bh-filter-bar">
+  <bh-filters
+    filters="DupeLotsCtrl.latestViewFilters"
+    on-remove-filter="DupeLotsCtrl.onRemoveFilter(filter)">
+  </bh-filters>
+</div>
+-->
+<div class="flex-content">
+  <div class="container-fluid">
+    <div
+      id="duplicate-lots-grid"
+      class="grid-full-height-with-filters"
```

Comment:
FYI - to make this grid full height, use the class `grid-full-height`.  This accounts for the "filter-bar".

Reply:
Since we don't have any filters on this registry, you should include this change.

Reply:
Done